fbw-common/src/wasm/systems/systems/src/communications/audio_control_panel.rs

```diff
@@ -0,0 +1,502 @@
+                    IdentificationWordAMUACP::Word01,
+                    1,
+                    LabelWordAMUACP::Label210VolumeControlVHF,
+                ),
+                WordAMUACPInfo::new(
+                    IdentificationWordAMUACP::Word02,
+                    2,
+                    LabelWordAMUACP::Label210VolumeControlVHF,
+                ),
+                WordAMUACPInfo::new(
+                    IdentificationWordAMUACP::Word03,
+                    3,
+                    LabelWordAMUACP::Label210VolumeControlVHF,
+                ),
+                WordAMUACPInfo::new(
+                    IdentificationWordAMUACP::Word04,
+                    1,
+                    LabelWordAMUACP::Label211VolumeControlHF,
+                ),
+                WordAMUACPInfo::new(
+                    IdentificationWordAMUACP::Word05,
+                    2,
+                    LabelWordAMUACP::Label211VolumeControlHF,
+                ),
+                WordAMUACPInfo::new(
+                    IdentificationWordAMUACP::Word06,
+                    1,
+                    LabelWordAMUACP::Label215VolumeControlINTCAB,
+                ),
+                WordAMUACPInfo::new(
+                    IdentificationWordAMUACP::Word07,
+                    2,
+                    LabelWordAMUACP::Label215VolumeControlINTCAB,
+                ),
+                WordAMUACPInfo::new(
+                    IdentificationWordAMUACP::Word08,
+                    3,
+                    LabelWordAMUACP::Label212VolumeControlADFPA,
+                ),
+                WordAMUACPInfo::new(
+                    IdentificationWordAMUACP::Word09,
+                    1,
+                    LabelWordAMUACP::Label213VolumeControlVORMKR,
+                ),
+                WordAMUACPInfo::new(
+                    IdentificationWordAMUACP::Word10,
+                    2,
+                    LabelWordAMUACP::Label213VolumeControlVORMKR,
+                ),
+                WordAMUACPInfo::new(
+                    IdentificationWordAMUACP::Word11,
+                    3,
+                    LabelWordAMUACP::Label213VolumeControlVORMKR,
+                ),
+                WordAMUACPInfo::new(
+                    IdentificationWordAMUACP::Word12,
+                    0,
+                    LabelWordAMUACP::Label217VolumeControlILS,
+                ),
+                WordAMUACPInfo::new(
+                    IdentificationWordAMUACP::Word15,
+                    1,
+                    LabelWordAMUACP::Label212VolumeControlADFPA,
+                ),
+                WordAMUACPInfo::new(
+                    IdentificationWordAMUACP::Word16,
+                    2,
+                    LabelWordAMUACP::Label212VolumeControlADFPA,
+                ),
+            ]),
+            last_complete_cycle_sent: Duration::from_millis(0),
+        }
+    }
+
+    fn send_word_0(&self, bus_acp: &mut Vec<Arinc429Word<u32>>) {
```

Comment:
Why do we need a `Vec` for the bus? Isn't a single arinc word enough?

Reply:
We could but the protocol documentation states that all the words will be sent every 170ms at an interval of 10ms even without any update on the audio panels.

As the simulation takes around 20ms, I chose to use a Vec to mitigate responsiveness.